src/api/controllers/followController.js

```diff
@@ -0,0 +1,57 @@
+import models from '../models/index';
+import status from '../../helpers/constants/status.codes';
+import errorSender from '../../helpers/error.sender';
+/**
+ * @class
+ */
+export default class Follow {
+  /**
+   * This function add the follow count to the database
+   * @param {*} req request sent by user
+   * @param {*} res response after excution of request
+   * @memberof Follow
+   * @return {object} res
+   */
+  static async follow(req, res) {
+    const { username } = req.params;
+    const follower = req.user.username;
+    try {
+      await models.Follow.create({
+        follower,
+        following: username
+      });
+      res.status(status.CREATED).json({
+        status: 201,
+        message: `You are now following ${username}`
+      });
+    } catch (error) {
+      errorSender(status.BAD_REQUEST, res, 'follow', `Already following ${username}`);
+    }
+  }
+
+  /**
+ * This function creates the unfollow process by deleting the user
+ * @param {*} req request sent by user
+ * @param {*} res response after execution of request
+ * @memberof Follow
+ * @returns {object} res
+ */
+  static async unfollow(req, res) {
+    const { username } = req.params;
+
+    const unfollowAct = await models.Follow.destroy({
+      where: {
+        following: username
```

Comment:
@truestbyheart  We only want to delete a specific record here, else we might delete all a user's followers.